doc/source/API/SetupTemplatesMaxwell.rst

```diff
@@ -8,19 +8,18 @@ You can edit a setup after it is created. Here is an example:
 
 .. code:: python
 
-    from pyaedt import Hfss
+    from pyaedt import Maxwell3d
 
-    hfss = Hfss()
+    Maxwell3d = Maxwell3d ()
     # Any property of this setup can be found on this page.
-    setup = hfss.create_setup()
-    setup.props["AdaptMultipleFreqs"] = True
-    setup.update()
-
-
+    setup = Maxwell3d.create_setup ()
+    setup.props["MaximumPasses"] = 5
+    setup.update ()
 
 .. pprint:: pyaedt.modules.SetupTemplates.MaxwellTransient
 .. pprint:: pyaedt.modules.SetupTemplates.Magnetostatic
 .. pprint:: pyaedt.modules.SetupTemplates.Electrostatic
 .. pprint:: pyaedt.modules.SetupTemplates.EddyCurrent
 .. pprint:: pyaedt.modules.SetupTemplates.ElectricTransient
-
+.. pprint:: pyaedt.modules.SetupTemplates.ACConduction
```

Comment:
@DaveTwyman please have a look at the doc build checks that fails. It says that in SetupTemplates there are no templates named "ACConduction" or "DCConduction".

Reply:
@gmalinve , Ok, they can be added in the future. Maybe at the same time as adjusting all the other solver templates that reference HFSS. I've removed them for now

Reply:
@DaveTwyman did you remove them?

Reply:
@gmalinve , I deleted the two lines in Pycharm and then hit commit.

Reply:
@DaveTwyman as you can see these lines are still here, let me try to do it on my side.